backend/views/core/currency_converter/dashboard.py

```diff
@@ -29,11 +37,58 @@ def convert_currency(init_currency, target_currency, amount):
     if not isinstance(amount, (int, float)):
         raise ValueError("Amount is not an accepted datatype")
```

Comment:
[Need data protection of currency and to stay one language and currency help](url)

Reply:
?